<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>examwidget</class>
 <widget class="QWidget" name="examwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#0055ff;&quot;&gt;考试管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextpushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>540</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>下一题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="handpushButton_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>540</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string> 交卷</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>700</width>
     <height>410</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>690</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QRadioButton" name="wrongradioButton_1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>错误</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>第1题</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_D_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AradioButton_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_C_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="CradioButton_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_A_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rightradioButton_1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_B_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="DradioButton_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BradioButton_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="answertextEdit_1">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>260</y>
        <width>650</width>
        <height>130</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="questiontextBrowser_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>650</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="valuelabel_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>（2分）</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QTextBrowser" name="questiontextBrowser_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>650</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="wrongradioButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>错误</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="CradioButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BradioButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AradioButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>第2题</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_B_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="answertextEdit_2">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>260</y>
        <width>650</width>
        <height>130</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rightradioButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="DradioButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_C_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_A_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_D_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="valuelabel_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>（2分）</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QTextBrowser" name="questiontextBrowser_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>650</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="wrongradioButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>错误</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="CradioButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BradioButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AradioButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>第3题</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_B_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="answertextEdit_3">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>260</y>
        <width>650</width>
        <height>130</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rightradioButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="DradioButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_C_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_A_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_D_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="valuelabel_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>（2分）</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QTextEdit" name="answertextEdit_4">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>260</y>
        <width>650</width>
        <height>130</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="DradioButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="wrongradioButton_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>错误</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_A_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BradioButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_B_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>第4题</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rightradioButton_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_C_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="questiontextBrowser_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>650</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_D_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AradioButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="CradioButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QLabel" name="valuelabel_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>（2分）</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page5">
     <widget class="QTextEdit" name="answertextEdit_5">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>260</y>
        <width>650</width>
        <height>130</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="DradioButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="wrongradioButton_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>错误</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_A_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BradioButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_B_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>第5题</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rightradioButton_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>正确</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_C_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="questiontextBrowser_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>650</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_D_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>590</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AradioButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="CradioButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QLabel" name="valuelabel_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>（2分）</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="sumlabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>本试卷共5题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backpushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>540</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>上一题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>考试剩余时间</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
